great_expectations/validator/metrics_calculator.py

```diff
@@ -41,6 +42,7 @@ def show_progress_bars(self) -> bool:
     def show_progress_bars(self, enable: bool) -> None:
         self._show_progress_bars = enable
 
+    @public_api
```

Comment:
If we are adding this to the `@public_api`, I'd like to request a complete docstring

Reply:
Committed.